Gruntfile.js

```diff
@@ -94,6 +94,7 @@ grunt.initConfig({
 		qunit: [
 			"test/index.html",
 			"test/autostart.html",
+			"test/amd.html",
```

Comment:
There's no specific reason. It would be certainly better if possible to import this list from a central point, but it can't be done this way as browserstack.json is static, and not the best place to make Gruntfile import this list.